website/docs/components/table/advanced-table/partials/specifications/states.md

```diff
@@ -0,0 +1,6 @@
+## States
```

Comment:
There are also the cell focused state and focus states for the cell expand/collapse button.